messages/run-summary-viewer.md

```diff
@@ -0,0 +1,27 @@
+# summary.header
+
+Summary
+
+# summary.found-no-violations
+
+Found 0 violations.
+
+# summary.violations-total
+
+Found %d violation(s):
+
+# summary.violations-item
+
+%d %s severity violation(s) found
```

Comment:
```suggestion
Found %d %s severity violation(s). 
```

Reply:
Okay?  I can't tell if the "real" error will look okay, because I don't know what %d and %s will actually contain.  But if possible I'd like to keep the same format as the other errors.  If the resulting error will look strange, put it back as before, but please add a period.